backstage-plugin/plugins/open-dora/src/i18n.ts

```diff
@@ -0,0 +1,19 @@
+import i18next from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+// Import all translation files 
```

Comment:
Trailing whitespace should be deleted to have the correct formatting.
```suggestion
// Import all translation files
```